### ISSUE TITLE
action_log_probs重复计算

### ISSUE THREAD

**Issue by cdm114514** [OpenLLMAI/OpenRLHF#301]
actor在做inference的时候不是可以直接返回logits吗，为什么experience maker里面还重算了一次

**Comment by hijkzzz** [comment]
to ensure accuracy.

**Comment by cdm114514** [comment]
Thanks!
